src/store/tests/Bridge/Meilisearch/StoreTest.php

```diff
@@ -0,0 +1,208 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\AI\Store\Tests\Bridge\Meilisearch;
+
+use PHPUnit\Framework\Attributes\CoversClass;
+use PHPUnit\Framework\TestCase;
+use Symfony\AI\Platform\Vector\Vector;
+use Symfony\AI\Store\Bridge\Meilisearch\Store;
+use Symfony\AI\Store\Document\VectorDocument;
+use Symfony\Component\HttpClient\Exception\ClientException;
+use Symfony\Component\HttpClient\MockHttpClient;
+use Symfony\Component\HttpClient\Response\MockResponse;
+use Symfony\Component\Uid\Uuid;
+
+#[CoversClass(Store::class)]
+final class StoreTest extends TestCase
+{
+    public function testStoreCannotInitializeOnInvalidResponse(): void
```

Comment:
Remove all `void` return types in tests please

Reply:
Yes, not to mention that PHPStan doesn't like when return types are not specified 😄

Reply:
It's just, that it does not add any value in tests.

Reply:
Well, I get that there is no functional value, but same is true for the `public` keyword for test methods.

I get that in combination with supporting various versions and having merge conflicts for nothing but coding best practices is annoying for `symfony/symfony` - but we don't have that struggle here. so for me adding return type declarations is a best practice to opt-out for reasons, but i see no reason here.

i'd vote for keeping them in